Soneta.Platform.Developer/ItemTemplateWizard/Wizards/SonetaAddonProjectTemplateWizard.cs

```diff
@@ -25,22 +25,46 @@ public void ProjectFinishedGenerating(Project project)
                 ProcessConfigFiles(project);
             else
                 EnsureSuffix(project);
+            AddFilesToSolution();
+        }
+        void AddFilesToSolution()
+        {
+            var solutionItemsFolder = GetProjects(SolutionItemsFolderName).FirstOrDefault() ?? ((Solution2)dte.Solution).AddSolutionFolder(SolutionItemsFolderName);
+            foreach (var file in SolutionFiles)
+            {
+                solutionItemsFolder.ProjectItems.AddFromFile(file);
+            }
         }
-
         public void ProjectItemFinishedGenerating(ProjectItem projectItem)
         {
         }
 
         public void RunFinished()
         {
         }
-
+        List<string> SolutionFiles = new List<string>();
```

Comment:
prywatne pole z dużej litery?
powinno być _solutionFiles, dlaczego nie solutionFiles to na dłuższą dyskusję np na programistycznym

Reply:
trzymając się standardu formatowania w pliku i projekcie to raczej bez podkreślenia

Reply:
poprawione